vm/src/stdlib/os.rs

```diff
@@ -512,33 +577,128 @@ pub fn make_module(vm: &VirtualMachine) -> PyObjectRef {
          "st_size" => ctx.new_property(StatResultRef::st_size),
     });
 
-    py_module!(vm, "_os", {
-        "open" => ctx.new_rustfunc(os_open),
+    let fd_funcs: Vec<(&str, PyObjectRef, Option<bool>, Option<bool>, Option<bool>)> = vec![
+        // name, func_obj, supports_fd, supports_dir_fd, supports_follow_symlinks
+        ("open", ctx.new_rustfunc(os_open), None, Some(false), None),
+        // access Some Some None
+        // chdir Some, None None
+        // chflags Some, None Some
+        // chmod Some Some Some
+        // chown Some Some Some
+        // chroot Some None None
+        (
```

Comment:
This is not very readable in the current way. It is hard to remember what parameter is which. How about changing this to be a vector of:
```rust
struct FdFunc {
   name: String,
   func_obj: PyObjectRef,
   supports_fd: boolean,
   supports_dir_fd: boolean,
   supports_follow_symlinks: boolean,
}
```

Reply:
I fixed this and other comments, thanks!